packages/api/.api/api.json

```diff
@@ -0,0 +1,11 @@
+{
+  "version": "1.0",
+  "apis": [
+    {
+      "identifier": "petstore",
+      "source": "https://raw.githubusercontent.com/readmeio/oas-examples/main/3.0/json/petstore.json",
+      "integrity": "sha512-FSapRqUaxZYIPMPT+DN0E6xUfVdD/WadjTacNvwrk7scTzXhTZGNReEQ/lZfg9cz8yIvec2fqhTUAF2L4jVfRQ==",
+      "installerVersion": "4.2.0"
```

Comment:
I feel like this should be something else, but I don't know what to change it to 😬

Reply:
Nah this is fine for now, it maps to the verison of `api` that you used when you did the install and because my branch is still at 4.2.0 that's what was used. `installerVersion` isn't actually used anywhere else yet, but I can see it used in the future where if we bump `api` and you have an old installer version we might be like "hey we made a bunch of performance improvements since <version>,  you should reinstall your SDK!"